### PR TITLE
Create Data.json

### PR DESCRIPTION
**Pull request by agussuparno**

### CODE REVIEW

**Review by noopman — CHANGES_REQUESTED** [review]
Hello there! Please follow the instructions in the repo readme file, and please ask if you have additional questions!

**Comment by agussuparno** [comment]
I have already pull request.
Thanks

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by noopman** [comment]
Hello @agussuparno how are things going for Global Azure on your end? We have had on update on this PR for a while and therefor I am reaching out to ask if I can help you somehow? If you do not need this PR we will close it in a few days!

**Comment by agussuparno** [comment]
Thanks, Magnus for your support.
My problems are solved.

Regards,
Agus
________________________________

File: 2022/MugiPurwokerto/Data.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
```
Comment (noopman) [inline 2022/MugiPurwokerto/Data.json:5]: There needs to be a list here of the valid folder-maintainers!

File: 2022/MugiPurwokerto/Data.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
```
Comment (noopman) [inline 2022/MugiPurwokerto/Data.json:4]: There needs to be a list here of the valid folder-maintainers!

File: 2022/MugiPurwokerto/Data.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      "",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "{Microsoft User Group Indonesia (Mugi) Purwokerto}",
+    "Country": "{Indonesia}",
+    "Url": "https://mugipurwokerto.or.id/",
+    "SessionizeApiId": "{7751}",
```
Comment (noopman) [inline 2022/MugiPurwokerto/Data.json:14]: This is not a valid ApiId!

File: 2022/MugiPurwokerto/Data.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named {foldername}.png.",
+      "File": "_template.png"
+    },
+    "Locations": [
+      {
+        "Name": "{Mugi Purwokerto}",
```
Comment (noopman) [inline 2022/MugiPurwokerto/Data.json:21]: You should not have { } in your entries! Just the text.

File: 2022/MugiPurwokerto/Data.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        "Address": "{Jl. Adipura VI No. 228, Purwosari, Baturaden, Central Java Indonesia}",
```
Comment (noopman) [inline 2022/MugiPurwokerto/Data.json:22]: You should not have { } in your entries! Just the text.

File: 2022/MugiPurwokerto/Data.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "{Microsoft User Group Indonesia (Mugi) Purwokerto}",
+    "Country": "{Indonesia}",
+    "Url": "https://mugipurwokerto.or.id/",
+    "SessionizeApiId": "{7751}",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named {foldername}.png.",
+      "File": "_template.png"
```
Comment (noopman) [inline 2022/MugiPurwokerto/Data.json:17]: Would you like to add your community logo to the folder and reference it here?

File: 2022/MugiPurwokerto/Data.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "{Microsoft User Group Indonesia (Mugi) Purwokerto}",
```
Comment (noopman) [inline 2022/MugiPurwokerto/Data.json:11]: You should not have { } in your entries! Just the text.

File: 2022/MugiPurwokerto/Data.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    "Country": "{Indonesia}",
```
Comment (noopman) [inline 2022/MugiPurwokerto/Data.json:12]: You should not have { } in your entries! Just the text.

**Comment by noopman** [comment]
I am missing a file called index.md in your community folder! Please look at the other contributions and emulate something similar!

**Review by noopman — CHANGES_REQUESTED** [review]
You need to please look at the other community folders and also read this repo's reademe.md to see what it is you need to do! Thank you!

**Comment by agussuparno** [comment]
Okay, I will change it.

Thanks

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by noopman** [comment]
This was a leftover from last year. Closing it.

**Comment by agussuparno** [comment]
Hi Magnus,
I just pull request to github for Global Azure Indonesia 2023.
Could you please review it.

Thank you,

Agus Suparno

**Pull request closed without merging**